internal/fs/wrappers/monitoring.go

```diff
@@ -42,7 +54,20 @@ func init() {
 			Aggregation: view.Sum(),
 			TagKeys:     []tag.Key{tags.FSOp, tags.FSError},
 		},
-	); err != nil {
+		&view.View{
+			Name:        "fs/ops_error_count",
+			Measure:     opErrorCount,
+			Description: "The cumulative number of errors generated by file system operations",
+			Aggregation: view.Sum(),
+			TagKeys:     []tag.Key{tags.FSError},
```

Comment:
shouldn't we use FSOP tag instead of creating seperate tags for error and latencies?

Reply:
Yes, FSOP will be more appropriate as it shows error count or latency by each method